packages/block-editor/src/components/dark-editor-style/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import tinycolor from 'tinycolor2';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect } from '@wordpress/element';
+
+export default function DarkEditorStyle( ref ) {
+	useEffect( () => {
+		const { ownerDocument } = ref.current;
+		const editorStylesWrapper = ownerDocument.getElementsByClassName(
+			'editor-styles-wrapper'
```

Comment:
I don't like that it assumes the existence of a className. That said this className is indeed the one responsible for applying editor styles, so these things might be good  together a `EditorStylesWrapper` component maybe?

Reply:
By looking at the code of the VisualEditor component and the number of hooks applied there, it seems that there's a clear node for some kind of reusable component for the block editor wrapper. But it something we can implement separately.